app/controllers/slack_controller.rb

```diff
@@ -0,0 +1,12 @@
+class SlackController < ApplicationController
```

Comment:
Whoops, never mind. Found [this](https://github.com/zendesk/samson/blob/ben/slash-command/plugins/slack_webhooks/config/routes.rb).